backend/RestAPI/Startup.cs

```diff
@@ -105,7 +105,6 @@ public void ConfigureServices(IServiceCollection services)
             }
 
             app.UseMiddleware<LoggingMiddleware>();
-            //app.UseHttpsRedirection();
```

Comment:
General comment: I think we should keep this one in production. Right?

Reply:
lol, this is not the production branch. But yes